build/helper.ts

```diff
@@ -19,6 +19,14 @@ const reDocUrl: InstallOptions['reDocUrl'] = (fileName, header) => {
   return docs + fileName + (_header ? `#${_header}` : '')
 }
 
+const reWebTypesSource: InstallOptions['reWebTypesSource'] = (title) => {
+  const symbol = `El${title.replace(/-/g, ' ').replace(/^\w|\s+\w/g, (item) => {
```

Comment:
```suggestion
  const symbol = `El${title.replaceAll('-', ' ').replaceAll(/^\w|\s+\w/, (item) => {
```

Reply:
```js
.replaceAll(/^\w|\s+\w/,
```

Is the wrong way to use, it should be

```js
.replaceAll(/^\w|\s+\w/g,
```

It's the same as replace

Reply:
My mistake. You're right.